swayidle/swayidle.1.scd

```diff
@@ -0,0 +1,59 @@
+swayidle (1)
+
+# NAME
+
+swayidle - Idle manager for Wayland
+
+# SYNOPSIS
+
+*swayidle* [options] [events...]
+
+# OPTIONS
+
+*-h*
+	Show help message and quit.
+
+*-d*
+	Enable debug output.
+
+# DESCRIPTION
+
+swayidle listens for idle activity on your Wayland compositor and executes tasks
+on various idle-related events. You can specify any number of events at the
+command line.
+
+#EVENTS
+
+*timeout* <timeout> <timeout command> [resume <resume command>]::
+	Execute <timeout command> if there is no activity for <timeout> seconds.
```

Comment:
Also `_` should be used instead of `<` and `>` in the description: `Execute _timeout command_ if …`

Reply:
Done